internal/mic/mic.go

```diff
@@ -61,3 +64,35 @@ func (mici *micImpl) ApplyMIC(ctx context.Context, name, ns string, images []kmm
 
 	return nil
 }
+
+func (mici *micImpl) GetModuleImageSpec(micObj *kmmv1beta1.ModuleImagesConfig, image string) *kmmv1beta1.ModuleImageSpec {
```

Comment:
What does `micObj` needs to be a pointer? Looks like we are not modifying it in the method.

Reply:
I won't say it is always better. It is a trade of between spending time on copying the object and having more control making sure no one is updating an object it shouldn't had.

Anyway, I am OK with keeping it as a pointer.